### ISSUE TITLE
Add a sqlectron --version command line

### ISSUE THREAD

**Issue by jleclanche** [sqlectron/sqlectron-gui#361]
Right now that just opens sqlectron.

**Comment by maxcnunes** [comment]
Sqlectron I a desktop app. I don't think is possible to use it as a command line to see the version. But you can check the version in the initial loading screen or in the menu bar.

**Comment by jleclanche** [comment]
There's nothing that prevents desktop apps from having a --version flag :) It's good practice.

[14:13:00] adys@azura ~ % pcmanfm --version
pcmanfm 0.11.3
[14:13:07] adys@azura ~ % keepassxc --version
keepassxc 2.2.0
[14:13:10] adys@azura ~ % chromium --version
Chromium 61.0.3163.100 
[14:13:17] adys@azura ~ % discord --version
Discord 0.0.2

**Comment by jleclanche** [comment]
BTW this is super useful for maintainers.

**Comment by maxcnunes** [comment]
I have implemented this on the branch merged above. But there are some caveats:

* OSX: it doesn't work because is not possible to start an app like a binary (I have to use the tool `open` to start the app from the command line and `open -a /Applications/Sqlectron.app --args --version` seems not to work).
* Windows: It prints out the version. But looks like the command still running after that. I had to press `Enter` to finish it. You will understand when you have tested it. I would appreciate the help from someone with Windows environment to solve that.
* Linux: I have not tested it.
